data_steward/analytics/table_metrics/metrics_over_time/dictionaries_and_lists.py

```diff
@@ -96,7 +98,8 @@
     'drug_success': True,
     'sites_measurement': True,
     'visit_date_disparity': True,
-    'date_datetime_disparity': True
+    'date_datetime_disparity': True,
```

Comment:
It would be good to have both of these in a constants file as well, like 
```suggestion
    consts.DATE_TIME_DISPARITY: consts.TRUE,
```